src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -274,6 +274,8 @@ export default {
 		}
 	},
 	render: function(createElement) {
+		const first = this.$slots.default[0]
+		first.componentOptions.propsData.icon = this.rootIcon
```

Comment:
You might have to use Vue.set to make sure Vue is aware of any key creation/change

Reply:
Works without Vue.set already, but let's add it.